src/app.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+const fs = require('fs');
+
+function funcRead() {
+  const terminal = process.argv.slice(2);
+  const filePath = './src/' + terminal[0];
+  const newFilePath = './src/' + terminal[1];
+  const isFile = fs.existsSync(filePath);
+
+  if (isFile) {
+    const fileInfo = fs.readFileSync(filePath, 'utf8');
+
+    fs.writeFile(newFilePath, fileInfo, (err, data) => {
```

Comment:
The callback provided to 'fs.writeFile' handles the 'err' argument but it does nothing in case of an error. You should at least log out the error or throw it so that it can be handled upstream.